favicon/models.py

```diff
@@ -1,14 +1,13 @@
 from compat import python_2_unicode_compatible
 
+import sys
+
 from django.db import models
 from django.conf import settings
 from django.core.files.uploadedfile import InMemoryUploadedFile
 
 from PIL import Image
-try:
-    from StringIO import StringIO
-except ImportError:
-    from io import StringIO
+from six import BytesIO
```

Comment:
@bashu could you please use the [compat](https://github.com/arteria/django-compat/) methods, in this case https://github.com/arteria/django-compat/blob/master/compat/__init__.py#L406

Thanks.